conversations/models.py

```diff
@@ -0,0 +1,151 @@
+import uuid
+import json
+from collections import OrderedDict
+
+from django.db import models
+from django.core.urlresolvers import reverse
+from django.contrib.auth.models import User
+
+from model_utils import Choices
+from model_utils.models import TimeStampedModel
+
+from conference.models import Conference
```

Comment:
Why is this import separated from model_utils? Shouldn't they be together?

Reply:
model_utils is a 3rd party package while `conference` is an app within the project :)